.github/workflows/publish.yml

```diff
@@ -0,0 +1,14 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - v*

```

Comment:
Can we centralize these? These should be the same for every project. Same for HE.